ferret/explainers/explanation_speech/lime_speech_explainer.py

```diff
@@ -143,7 +140,7 @@ def compute_explanation(
             scores=scores,
             explainer=self.NAME + "+" + removal_type,
             target=targets if n_labels > 1 else targets,
-            audio_path=audio_path,
+            audio_path=audio,
```

Comment:
As in the other explainers: change keyword argument's name to `audio` in the `ExplanationSpeech` class.